src/Struct/AgenticCommerce/V1/Money.php

```diff
@@ -0,0 +1,74 @@
+<?php declare(strict_types=1);
+/*
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Shopware\PayPalSDK\Struct\AgenticCommerce\V1;
+
+use OpenApi\Attributes as OA;
+use Shopware\PayPalSDK\Struct\Struct;
+
+#[OA\Schema(
+    schema: 'paypal_agentic_commerce_v1_money',
+    required: ['currencyCode', 'value']
+)]
+class Money extends Struct
```

Comment:
just wondering, these common objects (also addresses etc.) already exist in  `V2\Common`. We could ask, whether they defined them themselves or if they are reusing existing schemas, therefore we could reuse them.